test/create-element-to-jsx-props.js

```diff
@@ -0,0 +1,4 @@
+function foo() {
+  var a = React.createElement(Foo, { foo: "bar", bar: this.state.baz });
+  var b = React.createElement("div", { foo: "bar", bar: this.state.baz });
```

Comment:
Can you add these two cases:

* `React.createElement(Foo, {foo: 'bar', ...someObject})` to `<Foo foo="bar" {...someObject}>` – where the order of properties should be preserved, of course.
* Convert children-less components to `<Foo />` instead of `<Foo></Foo>`